app/src/renderer/components/wallet/PageWallet.vue

```diff
@@ -14,8 +14,7 @@ tm-page(data-title="Wallet", :title="config.devMode ? '' : 'Wallet'")
     tm-list-item(
       :title="wallet.address"
       :btn="'Receive'"
-      :overflow="true"
-      @click.native="copy")
```

Comment:
Why was this removed?

Reply:
the ux for copying form inputs was unsatisfactory. we can discuss ways of improving this.

the short address component is on every page and includes functionality for copying to clipboard (and the tooltip). no more copying to clipboard triggering notify.